README.rst

```diff
@@ -169,20 +133,20 @@ build and run the tests:
 You can also run the tests with Valgrind (a memory-error checker) by passing
 ``-DRASCAL_TESTS_USE_VALGRIND=ON`` to ``cmake``.
 
-In addition to testing the behaviour of the code, the test suite also check
-for formatting compliance with clang-format 8.0 or higher and black packages
-(these dependencies are optional). To install these dependencies on Ubuntu:
+In addition to testing the behaviour of the code, the test suite also check for
+formatting compliance with clang-format 8.0 or higher and black packages (these
+dependencies are optional). To install these dependencies on Ubuntu:
 
 .. code:: shell
 
    sudo apt-get install clang-format-8
-   pip3 install black
+   pip install -r requirements/testing.txt
 
 Build Type
-^^^^^^^^^^
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Usually the underline for the rst header is no longer than the title itself, i.e. 

```rst
Build Type
^^^^^^^^^^
```

but if this renders fine it works for me.